src/frontend/next/src/pages/layouts/MDXPageBase.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * A base for building all of our MDX pages.  Any other "global"
+ * things that need to happen for a page should get added here,
+ * and then they will trickle down to the other MDX pages.
+ *
+ * Make sure any new MDX style pages use this to wrap all
+ * elements in their render function.
+ */
+
+import { makeStyles } from '@material-ui/core/styles';
+import SEO from '../../components/SEO';
+import AboutFooter from '../../components/AboutFooter';
+
+type MDXPageBaseProps = {
+  children: any;
+  title: string;
+};
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    fontFamily: 'Roboto',
+    '& h1': {
+      color: theme.palette.primary.main,
```

Comment:
this is a good spot to use that 'fancy' object destructuring we discussed not so long ago @chrispinkney